Sources/PackageDescription/Version+StringLiteralConvertible.swift

```diff
@@ -25,6 +32,14 @@ extension Version: ExpressibleByStringLiteral {
 
 extension Version {
 
+    init(_ version: Version) {
```

Comment:
see line 15 (https://github.com/apple/swift-package-manager/pull/596/files/0ed1e2920c7cc657ec9fc188b6c923795a76b9f9#diff-33614ac85ab8c229e4d321abcf5b22aeR15)

I could have inlined it there but I thought this was cleaner...